apps/nextra/pages/en/build/create-aptos-dapp.mdx

```diff
@@ -50,7 +50,7 @@ import { Tabs } from 'nextra/components'
 
 To create an Aptos dapp, open your terminal, `cd` into the directory you’d like to create the dapp in, and run the following command:
 
-<Tabs items={['npm', 'pnpm', 'yarn']}>
+<Tabs items={['npx', 'pnpx']}>
```

Comment:
Since the tabs are based on the package manager name i think we should keep it as is and remove yarn

```suggestion
<Tabs items={['npm', 'pnpm']}>
```

Reply:
initializing the tool is with `npx` or `pnpx`, imo the tab name should match the actual command name